tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/NewLiferayModuleProjectOp.java

```diff
@@ -222,7 +225,50 @@
     @Label( standard = "Properties" )
     ListProperty PROP_PROPERTYKEYS = new ListProperty( TYPE, "PropertyKeys" );
     ElementList<PropertyKey> getPropertyKeys();
-    
+
+    // *** Liferay Bundle ***
```

Comment:
Don't add these properties to the ModuleOp class.  Lets create a new Op class for new NewJspHookModuleOp